src/functions-and-arrays.js

```diff
@@ -1,19 +1,139 @@
 // Iteration #1: Find the maximum
+const maxOfTwoNumbers = (a,b) => {
+  if (a>b) {
+    return a;
+  } else if (a<b) {
```

Comment:
Maybe it would have been even easier to put just:

if (a > b) {
    return a;
} else {
    return b;
}

But good job 👍